packages/local_auth/local_auth/lib/local_auth.dart

```diff
@@ -12,14 +12,14 @@ import 'dart:async';
 
 import 'package:flutter/foundation.dart' show visibleForTesting;
 import 'package:flutter/services.dart';
+import 'package:local_auth_platform_interface/local_auth_platform_interface.dart';
+import 'package:local_auth_platform_interface/types/auth_messages.dart';
+import 'package:local_auth_platform_interface/types/auth_options.dart';
+import 'package:local_auth_platform_interface/types/biometric_type.dart';
```

Comment:
We should really make local_auth_platform_interface.dart re-export the types needed to use the interface. That can be done in a follow-up PR, or a quick precursor to this one, as you prefer.

Reply:
Precursor PR: 
- #5196